ga4gh/datamodel/__init__.py

```diff
@@ -112,9 +113,13 @@ def _setAccessTimes(self, directoryPath):
         directoryPath.
         """
         # ctime is in seconds, and we want milliseconds since the epoch
-        ctimeInMillis = int(os.path.getctime(directoryPath) * 1000)
-        self._creationTime = ctimeInMillis
-        self._updatedTime = ctimeInMillis
+        if os.path.exists(directoryPath):
```

Comment:
I think we should just remove the condition here --- it doesn't actually do anything since the directory will always exist.